rep-0149.rst

```diff
@@ -512,6 +512,25 @@ Commonly used license strings:
  - MIT
  - Mozilla Public License Version 1.1
 
+Attributes
+''''''''''
+
+  .. raw:: html
+
+    <font color="blue">
+
+ ``file="FILE"`` *(optional)*
+
+  A relative path from ``package.xml`` pointing the full license text.
+
+  Popular open source licenses require provision of the license text when a user
```

Comment:
Imo

> Popular ...

doesn't describe the set of licenses well which have that requirement. Instead I would suggest

> Some ...

Reply:
Thanks. I agree.